README.md

```diff
@@ -0,0 +1,44 @@
+[![Coverage Status](https://coveralls.io/repos/github/Mumbere7/quickCredit/badge.svg?branch=develop)](https://coveralls.io/github/Mumbere7/quickCredit?branch=develop)
+[![Build Status](https://travis-ci.com/Mumbere7/quickCredit.svg?branch=master)](https://travis-ci.com/Mumbere7/quickCredit)
```

Comment:
The build status badge should point to `develop` branch. Yours is pointing to `master`

Reply:
Okay I will do that.